smart_open/tests/test_smart_open.py

```diff
@@ -193,16 +194,28 @@ def test_open_with_keywords_explicit_r(self):
         self.assertEqual(expected, actual)
 
     @unittest.skipIf(
-        pkgutil.find_loader('pathlib') is None,
-        "do not test pathlib support if pathlib is not available")
+        (pkgutil.find_loader('pathlib') is None and
+         pkgutil.find_loader('pathlib2') is None),
+        "do not test pathlib support if pathlib or backport are not available")
     def test_open_and_read_pathlib_path(self):
-        """If pathlib.Path is available we should be able to open and read."""
-        from pathlib import Path
+        """If ``pathlib.Path`` is available we should be able to open and read."""
+        fpath = os.path.join(CURR_DIR, 'test_data/cp852.tsv.txt')
+
+        # Import ``pathlib`` if the builtin ``pathlib`` or the backport
+        # ``pathlib2`` are available. The builtin ``pathlib`` will be imported
+        # with higher precedence.
+        for pathlib_module in ('pathlib', 'pathlib2'):
+            try:
+                pathlib = importlib.import_module(pathlib_module)
+                break
+            # Unit test will skip if either module is unavailable so it's safe
+            # to assume we can import _at least_ one working ``pathlib``.
+            except ImportError:
+                pass
 
         # builtin open() supports pathlib.Path in python>=3.6 only
```

Comment:
I don't think we need to bother with this is in the tests. From the point of view of the test, it's an unnecessary detail. In this particular case, we just want to read the expected data in the quickest and simplest possible way. So in this case, I think it's better to just do:

```python
with open(fpath) as fin:
    expected = fin.read().decode('cp852')
```